xml/schema/Case/Case.xml

```diff
@@ -79,7 +79,7 @@
     <comment>Date on which given case starts.</comment>
     <html>
       <type>Select Date</type>
-      <formatType>activityDateTime</formatType>
+      `<formatType>activityDate</formatType>
```

Comment:
stray ` at the start of this line @mattwire

Reply:
Oh for goodness sake...